contributors/design-proposals/architecture/architecture.md

```diff
@@ -237,7 +237,7 @@ Service endpoints are found primarily via [DNS](https://kubernetes.io/docs/conce
 A number of components, called [*add-ons*](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons) typically run on Kubernetes
 itself:
 * [DNS](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns)
-* [Ingress controller](https://github.com/kubernetes/ingress/tree/master/controllers)
+* [Ingress controller](https://github.com/kubernetes/ingress/)
```

Comment:
Seems like it should be `https://github.com/kubernetes/ingress-nginx/tree/master/pkg/ingress/controller`?

Reply:
Looking at old link you're probably right, I changed it